### PR TITLE
Minor improvements to AWS EKS quickstart

### PR DESCRIPTION
**Pull request by vuldin**
- expand prerequisites to include `aws` and `eksctl` CLIs
- add info on required IAM permissions
- add command to update kubeconfig to point to new cluster

### CODE REVIEW

File: docs/platform/quickstart/kubernetes-qs-dev.mdx [hunk 1]
```diff
@@ -20,8 +20,8 @@ This page uses the latest recommended Helm chart for installing Redpanda. For in
 
 Verify that you have the following software installed:
 
-* [kubectl](https://kubernetes.io/docs/tasks/tools/) - version 1.21 or later
-* [Helm](https://github.com/helm/helm/releases) - version 3.0.0 or later
+* [kubectl](https://kubernetes.io/docs/tasks/tools/) - version 1.21 or later (note: the EKS route below uses Kubernetes version 1.22)
+* [Helm](https://helm.sh/docs/intro/install/) - version 3 or later (note: the EKS route below requires the [latest 3.8 version](https://github.com/helm/helm/releases/tag/v3.8.2))
```
Comment (Feediver1) [inline docs/platform/quickstart/kubernetes-qs-dev.mdx:24]: We have a note admonition you can use:

:::note
The EKS route below uses Kubernetes version 1.22
:::

 or, if you do not want an explicit note/admonition here, you can rewrite:
(note that the EKS route below uses Kubernetes version 1.22)

Same goes for the next bullet.

File: docs/platform/quickstart/kubernetes-qs-dev.mdx [hunk 2]
```diff
@@ -70,7 +70,13 @@ kubectl taint node \
   </TabItem>
   <TabItem value="eks" label="Amazon EKS">
 
-Create an Elastic Kubernetes Service (EKS) cluster:
+> There are two additional CLIs needed for creating a Kubernetes cluster in AWS:
+> 1. [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+> 2. [eksctl <= v0.114.0](https://github.com/weaveworks/eksctl/releases/tag/v0.114.0), more details on how to install [here](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
```
Comment (Feediver1) [inline docs/platform/quickstart/kubernetes-qs-dev.mdx:75]:
```suggestion
> 2. [eksctl <= v0.114.0](https://github.com/weaveworks/eksctl/releases/tag/v0.114.0). Install details available  [here](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
```

File: docs/platform/quickstart/kubernetes-qs-dev.mdx [hunk 3]
```diff
@@ -70,7 +70,13 @@ kubectl taint node \
   </TabItem>
   <TabItem value="eks" label="Amazon EKS">
 
-Create an Elastic Kubernetes Service (EKS) cluster:
+> There are two additional CLIs needed for creating a Kubernetes cluster in AWS:
+> 1. [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
```
Comment (Feediver1) [inline docs/platform/quickstart/kubernetes-qs-dev.mdx:74]:
```suggestion
1. [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
```

File: docs/platform/quickstart/kubernetes-qs-dev.mdx [hunk 4]
```diff
@@ -70,7 +70,13 @@ kubectl taint node \
...
+> 2. [eksctl <= v0.114.0](https://github.com/weaveworks/eksctl/releases/tag/v0.114.0), more details on how to install [here](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+
+> You must ensure that appropriate IAM permissions are applied to the user who will be running the following commands. More details can be found [here](https://docs.aws.amazon.com/eks/latest/userguide/security_iam_id-based-policy-examples.html).
```
Comment (Feediver1) [inline docs/platform/quickstart/kubernetes-qs-dev.mdx:77]:
```suggestion
You must ensure that appropriate IAM permissions are granted to the user who will be running the following commands. More details can be found [here](https://docs.aws.amazon.com/eks/latest/userguide/security_iam_id-based-policy-examples.html).
```

File: docs/platform/quickstart/kubernetes-qs-dev.mdx [hunk 5]
```diff
@@ -80,6 +86,14 @@ eksctl create cluster --name redpanda \
     --nodes-min 3 \
     --nodes-max 4
 ```
+
+Once the cluster is created, make sure your local kubeconfig is updated to point to this cluster:
```
Comment (Feediver1) [inline docs/platform/quickstart/kubernetes-qs-dev.mdx:90]:
```suggestion
After the cluster is created, be sure to update your local kubeconfig to point to this cluster:
```

**Review by Feediver1 — APPROVED** [review]
Approved with updates

**Comment by JakeSCahill** [comment]
This seems to fix #765

**Pull request merged**